js/ez-browse.js

```diff
@@ -0,0 +1,66 @@
+(function () {
+    /**
+     * <ez-browse> represents a clickable element in the application which will fire
+     * the `ez-contentDiscover` event on click. This event is responsible for launching
+     * the universal discovery widget component. Some config and listeners are given to the event
+     * Like the selected location id which will determine the starting location of the UDW and
+     * a listnener on the `ez-confirm` event which will browse to a given location with the url
+     * provided by the location.
+     */
+    class Browse extends Polymer.Element {
+        static get is() {
+            return 'ez-browse';
+        }
+
+        static get properties() {
+            return {
+                'selectedLocationId': {
+                    type: String,
+                },
+            };
+        }
+
+        constructor() {
+            super();
+            this.addEventListener('click', this._discoverContent.bind(this));
+        }
+
+        /**
+         * Dispatches `ez:contentDiscover` event with config and listeners
+         */
+        _discoverContent() {
+            const event = new CustomEvent('ez:contentDiscover', {
+                detail: {
+                    config: {
+                        title: 'Browse',
+                        startingLocationId: this.selectedLocationId !== 'false' ? this.selectedLocationId : undefined,
```

Comment:
`selectedLocationId` can only be a Number or undefined, you don't even need that (wrong) condition